mount/mount_linux.go

```diff
@@ -271,21 +367,16 @@ func (mounter *SafeFormatAndMount) formatAndMount(source string, target string,
 
 	if !readOnly {
 		// Run fsck on the disk to fix repairable issues, only do this for volumes requested as rw.
```

Comment:
This comment is not accurate.  Run fsck or xfs_repair on the disk to fix repairable issues based on file system type